src/Entities/Nodes.jl

```diff
@@ -89,28 +89,15 @@ function Base.show(io::IO, ::MIME"text/plain", n::Node)
     println("• Node at $(n.x) and dofs $(n.dofs)")
 end
 
-"1D node constructor with one dim coordinates."
-function Node(x₁::T, dofs::Dictionary=Dictionary{Field,Vector{Dof}}()) where {T<:Real}
-    Node(Point(x₁), dofs)
-end
-
-"2D node constructor with two coordinates."
-function Node(x₁::T, x₂::T, dofs::Dictionary=Dictionary{Field,Vector{Dof}}()) where {T<:Real}
-    Node(Point(x₁, x₂), dofs)
-end
-
-"3D node constructor with coordinates."
-function Node(x₁::T, x₂::T, x₃::T,
-              dofs::Dictionary=Dictionary{Field,Vector{Dof}}()) where {T<:Real}
-    Node(Point(x₁, x₂, x₃), dofs)
-end
-
 "Node constructor with a `NTuple`."
 function Node(t::NTuple{dim,T},
               dofs::Dictionary=Dictionary{Field,Vector{Dof}}()) where {dim,T<:Real}
     Node(Point(t), dofs)
 end
 
+"`Node` constructor to promote types of coordinates of Point"
+Node(t::Union{Tuple, AbstractArray}, dofs::Dictionary=Dictionary{Field, Vector{Dof}}()) = Node(promote(t...), dofs)
```

Comment:
The problem is that `t` or usually `args`  type is just a plain: 

```julia
typeof(t) = Tuple{Int64, Int64, Int64, Dictionary{Symbol, Vector{Int64}}}
```
so one can't do a dispatch getter like we tried. Given that, I think what we want is: 
```suggestion
function Node(t)
    dofs = if length(t) > 1 && t[end] isa Dictionary
        t[end]
    else
        Dictionary{Field,Vector{Dof}}()
    end
    coords = t[1:(end - 1)]
    Node(promote(coords)..., dofs)
end

```

Reply:
Also I suggest you to apply my changes directly clicking on "commit suggestion" which is most strigthiforwd way to do it.  Please take a look at https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request